runtimebp/cpu.go

```diff
@@ -48,52 +39,17 @@ func NumCPU() (n float64) {
 		return
 	}
 
-	period, err = readNumberFromFile(periodPath, buf)
-	if err != nil {
+	// CFS quota returns -1 if there is no limit, return the default.
+	if quota < 0 {
 		return
```

Comment:
Are named return values common elsewhere in baseplate.go? Personally favor explicit returns but not sure if we have an "official" style recommendation here 🤷‍♂️

Reply:
I wouldn't say they are super common, we mostly use them in monitoring contexts.  That said, I think we've been trying to encourage everyone to still have the return be explicit when we have named returns but this predates us being more deliberate about that 😅

Reply:
there are two different things here: named return and naked return (while naked return is only possible with named return).

named return is necessary when you need to access (and optionally, modify) the return value, and we use that pattern a lot in middlewares. named return is also useful for documentation purpose (when the return type is not enough to infer the meaning of the return value).

naked return is in general not recommended, this code was just written early enough to predate that style guide 😅

Reply:
Done, removed named return / naked returns.